src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayCellModel.swift

```diff
@@ -115,28 +118,88 @@ final class DiaryOverviewDayCellModel {
 			return nil
 		}
 	}
-
+	
+	var hideCheckinRisk: Bool {
+		return checkinsWithRisk.isEmpty
+	}
+	
+	var checkinImage: UIImage? {
+		if checkinsWithRisk.isEmpty {
+			return nil
+		} else {
+			return checkinsWithRisk.contains(where: { $0.risk == .high }) ?
+				UIImage(imageLiteralResourceName: "Icons_Attention_high") :
+				UIImage(imageLiteralResourceName: "Icons_Attention_low")
+		}
+	}
+	
+	var checkinTitleHeadlineText: String? {
+		if checkinsWithRisk.isEmpty {
+			return nil
+		} else {
+			return checkinsWithRisk.contains(where: { $0.risk == .high }) ?
+				AppStrings.ContactDiary.Overview.CheckinEncounter.titleHighRisk :
+				AppStrings.ContactDiary.Overview.CheckinEncounter.titleLowRisk
+		}
+	}
+	
+	var checkinTitleAccessibilityIdentifier: String? {
+		if checkinsWithRisk.isEmpty {
+			return nil
+		} else {
+			return checkinsWithRisk.contains(where: { $0.risk == .high }) ?
+				AccessibilityIdentifiers.ContactDiaryInformation.Overview.checkinRiskLevelHigh :
+				AccessibilityIdentifiers.ContactDiaryInformation.Overview.checkinRiskLevelLow
+		}
+	}
+	
+	var checkinDetailDescription: String? {
+		return checkinsWithRisk.isEmpty ? nil: AppStrings.ContactDiary.Overview.CheckinEncounter.titleSubheadline
+	}
+	
+	var isSinlgeRiskyCheckin: Bool {
+		return checkinsWithRisk.count > 1 ? false : true
```

Comment:
Binary operator?
```suggestion
		return checkinsWithRisk.count <= 1
```

Reply:
The zero '0' case is odd… to be perfectly fine this should be an enum with an 'empty' state :)

Reply:
I think it is sufficient to check for 1 or more elements because its only to know if we add the suffix or not.